### DISCUSSION TITLE
Performance optimizations

### DISCUSSION THREAD

**Discussion by underfin** [rolldown/rolldown#875] [Core Minutes]
Some optimizations that we want to do in the future.

Each optimization is posted in single comment. Discussions of each optimization should be limited to that comment.

**Comment by hyfdev** [comment]
Drop larger memory in another thread by @underfin 

<img width="1055" alt="image" src="https://github.com/rolldown/rolldown/assets/14008915/b44040af-c55e-4696-9ca5-18a598669052">

ref: https://github.com/rolldown/rolldown/pull/842

**Comment by hyfdev** [comment]
Store `include!('runtime.js')` in `OnceCell` and wrapped it with `Arc`.

This helps saving memory while there are many times of bundling.

**Comment by hyfdev** [comment]
Warmup files

ref: https://github.com/rolldown/rolldown/discussions/875
